lightkurve/search.py

```diff
@@ -419,8 +407,12 @@ def search_targetpixelfile(target, radius=None, cadence='long',
         units of arcseconds.  If `None` then we default to 0.0001 arcsec.
     cadence : str
         'long' or 'short'.
-    mission : str, list of str
+    mission : str, tuple of str
         'Kepler', 'K2', or 'TESS'. By default, all will be returned.
+    author : str, tuple of str, or "any"
```

Comment:
Passing `any` returns K2SC light curves, which don't seem to be supported. They will be downloaded but not fail when `read` is called on them. 

```
ValueError: Not recognized as a Kepler or TESS data product
```

We should either make it possible to open these in `lightkurve` or remove them from the list for now.